Cargo.toml

```diff
@@ -24,8 +25,8 @@ chrono = "0.4.38"
 clap = "4.5.20"
 config = "0.14.1"
 dcap-qvl = "0.1.6"
-fastcrypto = { git = "https://github.com/MystenLabs/fastcrypto", rev="69d496c71fb37e3d22fe85e5bbfd4256d61422b9", package = "fastcrypto" }
-fastcrypto-zkp = { git = "https://github.com/MystenLabs/fastcrypto", rev="69d496c71fb37e3d22fe85e5bbfd4256d61422b9", package = "fastcrypto-zkp" }
+fastcrypto = { git = "https://github.com/MystenLabs/fastcrypto", rev = "69d496c71fb37e3d22fe85e5bbfd4256d61422b9", package = "fastcrypto" }
```

Comment:
This is very minor but I suppose we may want to commit a `.editorconfig` so that we have consistent formatting here